pkg/printers/internalversion/printers.go

```diff
@@ -825,7 +839,12 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 		reason = "Terminating"
 	}
 
-	row.Cells = append(row.Cells, pod.Name, fmt.Sprintf("%d/%d", readyContainers, totalContainers), reason, int64(restarts), translateTimestampSince(pod.CreationTimestamp))
+	lastRestart := "<none>"
+	if !lastRestartDate.IsZero() {
```

Comment:
No need for this condition, `translateTimestampSince` will return `<unknown>` for zero value.

Reply:
Good to know, thank you!